api/google-manager/services/google-manager.js

```diff
@@ -6,33 +6,73 @@ const uuid = require('uuid/v4');
 // Doc is at https://googleapis.dev/nodejs/googleapis/latest/index.html
 
 const { isDevEnv } = require('../../../utils/isDevEnv');
+const { v4: uuidv4 } = require('uuid');
 
 class GoogleManager {
-  constructor(email, key, groupID, serverBaseURL, auditFreqMins) {
+  constructor(email, key, serverBaseURL, auditFreqMins) {
     const scopes = [
```

Comment:
Should we rename this to something like `adminScope`?

Reply:
sure!